packages/blockchain-wallet-v4-frontend/src/scenes/Settings/General/LinkedBanks/index.tsx

```diff
@@ -78,7 +78,7 @@ class LinkedBanks extends PureComponent<Props> {
 const mapStateToProps = (state: RootState): LinkStatePropsType => ({
   data: getData(state),
   fiatCurrency: selectors.components.withdraw.getFiatCurrency(state),
-  walletCurrency: selectors.core.settings.getCurrency(state)
+  walletCurrency: selectors.core.settings.getCurrency(state).getOrElse('USD')
```

Comment:
We really need to remove the need for this.

Reply:
absolutely 💯